adal/src/main/java/com/microsoft/aad/adal/SSOStateSerializer.java

```diff
@@ -62,15 +62,18 @@
             .registerTypeAdapter(TokenCacheItem.class, new TokenCacheItemSerializationAdapater())
             .create();
 
-    private int getVersion() {
-        return version;
+    /**
+     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+     */
+    @SuppressWarnings("unused")
```

Comment:
I'm just trying to figure out what's changed. This constructor was private since the first day we code it up, trying to figure out what is the test that's using a private constructor.